model/components/prometheus.yml

```diff
@@ -0,0 +1,105 @@
+---
+components:
+
+  # prometheus
+  - id: prometheus
+    name: Prometheus
+    description: Prometheus is an open-source systems monitoring and alerting toolkit.
+    link: https://prometheus.io/
+    logo: prometheus
+    categories:
+      - agent
+    capabilities:
+      - tech:
+        - java
+        - go
+        - scala
+        - python
+        - ruby
+        - bash
+        - c++
+        - common-lisp
+        - elixir
+        - erlang
+        - haskell
+        - lua
+        - .net
+        - c#
+        - nodejs
+        - php
+        - rust
+      - aspects:
+        - metrics-app
+        - metrics-process
+
+  # prometheus Server
+  - id: prometheus-server
+    name: Prometheus Server
+    description: The Jaeger collector receives traces from Jaeger agents and runs them through a processing pipeline. Currently our pipeline validates traces, indexes them, performs any transformations, and finally stores them.
+    link: https://prometheus.io/
+    logo: prometheus
+    categories:
+      - collector
+      - visualization
+      - storage
+    connections:
+      - graphite
+      - influx-db
+      - chronix
+      - weavecortex
+      - crate-db
+      - open-tsdb
+      - postgresql
+      - timescala-db
+      - grafana
+      - prometheus-exporter
+      - prometheus
+
+  # prometheus alerting
+  - id: prometheus-alerting
+    name: Prometheus Alert Manager
+    description: The Alertmanager handles alerts sent by client applications such as the Prometheus server. It takes care of deduplicating, grouping, and routing them to the correct receiver integration such as email, PagerDuty, or OpsGenie. It also takes care of silencing and inhibition of alerts.
+    link: https://prometheus.io/
+    logo: prometheus
+    categories:
+      - alerting
+    connections:
+      - prometheus-server
+      - pager-duty
```

Comment:
I would remove the notification channels for now. Not sure if we should handle them in same way as other tools. OK?

Reply:
I would agree, if we would have them in the model, then not as connections